tasks/cea-install.js

```diff
@@ -59,15 +59,16 @@ const npmInit = async () => {
 
 // Copy root and src files to target directory
 const srcInit = async () => {
-  const copyBlacklist = ['tasks/'];
-  // TODO .gitignore missing https://github.com/ProjectEvergreen/create-evergreen-app/issues/59
+  const copyBlacklist = ['tasks/', '.gitignore'];
```

Comment:
Hmm, I think maybe it just makes sense to remove _.gitignore_ from `files` then in _package.json_?  It's not as if it would exist at the time this script is running.

Not sure, what do you think?

Reply:
That works too. Will do that then.